qualification/general/test_control.py

```diff
@@ -236,6 +227,10 @@ async def test_control(
     pdf_report.detail(f"Subscribed to {len(sensor_watcher.sensors)} sensors.")
 
     pdf_report.step("Set up periodic control.")
+    # We don't actually care about the gains, but setting it updates the
+    # steady-state timestamp, and ensures that consume_chunks sees only
+    # "fresh" data.
+    await cbf.product_controller_client.request("gain-all", "antenna-channelised-voltage", "default")
```

Comment:
Might be unnecessary after NGC-1462?

Reply:
No, that's unrelated. The fixture setup will listen to TACV heaps until it gets a good one, and during that time, BCP heaps will also be received and go into the ringbuffer. We want to ignore those.